pymc/distributions/continuous.py

```diff
@@ -2501,8 +2501,8 @@ class HalfStudentTRV(RandomVariable):
     _print_name = ("HalfStudentT", "\\operatorname{HalfStudentT}")
 
     @classmethod
-    def rng_fn(cls, rng, nu, sigma, size=None) -> np.ndarray:
-        return np.asarray(np.abs(stats.t.rvs(nu, sigma, size=size, random_state=rng)))
+    def rng_fn(cls, rng, nu, scale=None, size=None) -> np.ndarray:
+        return np.asarray(np.abs(stats.t.rvs(nu, scale, size=size, random_state=rng)))
```

Comment:
Should be something like this.

```suggestion
    def rng_fn(cls, rng, nu, sigma, size=None) -> np.ndarray:
        return np.abs(stats.t.rvs(nu, scale=sigma, size=size, random_state=rng))
```

Hopefully one of our existing tests now fails and will need to be updated. 

Otherwise it means this functionality was not covered by our existing tests and we need a new test or improve the existing ones.

Reply:
Hey @ricardoV94 I have made changes according to the suggestions. Hopefully the tests pass but if they don't, I'll work on adding the tests. Also from what I understand, the changes here might not be able to eliminate the errors raised in the discussion [here](https://discourse.pymc.io/t/pm-halfstudentt-dist-failing-to-give-correct-shape/11821/5)  and [here](https://discourse.pymc.io/t/pm-halfstudentt-dist-failing-to-give-correct-shape/11821/6) where we want to pass keyword ```scale``` instead of ```sigma``` to the ```pm.HalfStudentT```.

Reply:
No, the idea is not to pass scale. The problem was sigma was being passed as loc (which is the second unnamed argument) to the scipy function by mistake. It should have been passed as scale, as this PR does.